samples/sdfw/ssf_client/src/main.c

```diff
@@ -70,6 +71,21 @@ static void sdfw_update(void)
 	}
 }
 
+static void device_info(void)
+{
+	int err;
+
+	char uuid_value[41] = "00000000-0000-0000-0000-000000000000\0";
+	int uuid_value_len = strlen(uuid_value);
```

Comment:
UUID length is always the same. So I see no benefit in getting the length because it always returns `36` here.

Reply:
this has been reworked.